WordPress/Classes/ViewRelated/NUX/Signin2FAViewController.swift

```diff
@@ -41,6 +41,8 @@ import WordPressShared
         configureSendCodeButtonText()
         configureStatusLabel("")
         configureSubmitButton(animating: false)
+        
+        self.sourceTag = SupportSourceTag.wpComLogin
```

Comment:
I know this was already merged — just want to note that I don't think it's necessary to specify the two factor screen. Users are pretty clear when that's their problem, and in terms of tracking how many issues arise, I don't think that needs to be a priority for auto-tagging issues right now.